classes/RightSignature.php

```diff
@@ -77,9 +77,9 @@ public function prepackageTemplate($templateGuid, $callbackUrl=null)
 	 * @param int $documentGuid document GUID
 	 * @return RightSignature\Document
 	 */
-	public function document($documentGuid)
+	public function documentDetails($documentGuid)
```

Comment:
A deprecated method for `document` that points to this one would be a good way to not break older versions.